src/Resources/config/services.xml

```diff
@@ -14,27 +14,20 @@
         <parameter key="ruudk_payment_mollie.form.banktransfer_type.class">Ruudk\Payment\MollieBundle\Form\BanktransferType</parameter>
         <parameter key="ruudk_payment_mollie.form.belfius_type.class">Ruudk\Payment\MollieBundle\Form\BelfiusType</parameter>
         <parameter key="ruudk_payment_mollie.form.kbc_type.class">Ruudk\Payment\MollieBundle\Form\KbcType</parameter>
+        <parameter key="ruudk_payment_mollie.form.inghomepay_type.class">Ruudk\Payment\MollieBundle\Form\InghomepayType</parameter>
+        <parameter key="ruudk_payment_mollie.form.bitcoin_type.class">Ruudk\Payment\MollieBundle\Form\BitcoinType</parameter>
+        <parameter key="ruudk_payment_mollie.form.paypal_type.class">Ruudk\Payment\MollieBundle\Form\PaypalType</parameter>
+        <parameter key="ruudk_payment_mollie.form.paysafecard_type.class">Ruudk\Payment\MollieBundle\Form\PaysafecardType</parameter>
         <parameter key="ruudk_payment_mollie.plugin.default.class">Ruudk\Payment\MollieBundle\Plugin\DefaultPlugin</parameter>
         <parameter key="ruudk_payment_mollie.plugin.ideal.class">Ruudk\Payment\MollieBundle\Plugin\IdealPlugin</parameter>
         <parameter key="ruudk_payment_mollie.api_key" />
-        <parameter type="collection" key="ruudk_payment_mollie.ideal.issuers.test">
-            <parameter key="ideal_ABNANL2A">ABN AMRO (test)</parameter>
-            <parameter key="ideal_ASNBNL21">ASN Bank (test)</parameter>
-            <parameter key="ideal_BUNQNL2A">Bunq (test)</parameter>
-            <parameter key="ideal_INGBNL2A">ING (test)</parameter>
-            <parameter key="ideal_KNABNL2H">Knab (test)</parameter>
-            <parameter key="ideal_RABONL2U">Rabobank (test)</parameter>
-            <parameter key="ideal_RBRBNL21">RegioBank (test)</parameter>
-            <parameter key="ideal_SNSBNL2A">SNS Bank (test)</parameter>
-            <parameter key="ideal_TRIONL2U">Triodos Bank (test)</parameter>
-            <parameter key="ideal_FVLBNL22">Van Lanschot (test)</parameter>
-        </parameter>
-        <parameter type="collection" key="ruudk_payment_mollie.ideal.issuers.live">
+        <parameter type="collection" key="ruudk_payment_mollie.ideal.issuers">
             <parameter key="ideal_ABNANL2A">ABN AMRO</parameter>
             <parameter key="ideal_ASNBNL21">ASN Bank</parameter>
-            <parameter key="ideal_BUNQNL2A">Bunq</parameter>
+            <parameter key="ideal_BUNQNL2A">bunq</parameter>
```

Comment:
Can you please revert this line? I really don't like this. Every brandname should start with a capital.

Reply:
People that want to have it starting with a lowercase b are free to change that / overwrite it.